assets/training/finetune_acft_hf_nlp/src/register_model/register_model.py

```diff
@@ -149,6 +166,9 @@ def convert_lora_weights_to_safetensors(model_path: str):
         os.remove(bin_file)
         logger.info(f"Deleted {bin_file}")
 
+    # update adapter_config.json with `peft_version`
+    update_peft_adapter_config(args.model_path)
```

Comment:
can we call this function inside `if __name__ == "__main__"`  as well

`convert_lora_weights_to_safetensors` does not say that it will update adapter config as well

Reply:
done